### ISSUE TITLE
Firebase with Discord Bot

### ISSUE THREAD

**Issue by Aliyss** [RunEMC/NootBot#1]
Hey yo,

I just wanted to ask, if you knew how to deploy a discord bot in node.js to firebase and if that would include hosting the bot for free. I'm currently using Heroku, but I wanted to manage databases. I would very much appreciate an answer.

Merci for letting me have a look into NootBot.
AS
SB

**Comment by RunEMC** [comment]
Hi Aliyss, It's been awhile since I've worked on Nootbot and it's been more of a learning project for me more than anything. I've just recently built a new desktop so I've been trying to get the cloned repo working on my new pc, but I've made quite a few mistakes in making this bot which I am trying to fix. If I get firebase to work, I'll let you know, but I'm currently working on some other projects right now and it might be awhile before I get back to this.
